src/cmd/singularity/actions.go

```diff
@@ -14,8 +14,8 @@ import (
 	"strings"
 
 	"github.com/opencontainers/runtime-spec/specs-go"
-	config "github.com/singularityware/singularity/internal/pkg/runtime/engine/singularity/config"
-	"github.com/singularityware/singularity/pkg/configs"
+	runtimeconfig "github.com/singularityware/singularity/src/runtime/workflows/workflows/singularity/config"
```

Comment:
Nested `worklfows/workflows` is going to get confusing - is there any other sensible naming here?